addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime, timedelta
+
+from odoo import fields
+from odoo import Command
+from odoo.addons.event_booth_sale.tests.common import TestEventBoothSaleCommon
+from odoo.tests.common import users
+
+
+class TestEventBoothSale(TestEventBoothSaleCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestEventBoothSale, cls).setUpClass()
+
+        cls.event_0 = cls.env['event.event'].create({
+            'name': 'TestEvent',
+            'auto_confirm': True,
+            'date_begin': fields.Datetime.to_string(datetime.today() + timedelta(days=1)),
+            'date_end': fields.Datetime.to_string(datetime.today() + timedelta(days=15)),
+            'date_tz': 'Europe/Brussels',
+        })
+
+        cls.booth_1 = cls.env['event.booth'].create({
+            'name': 'Test Booth 1',
+            'booth_category_id': cls.event_booth_category_1.id,
+            'event_id': cls.event_0.id,
+        })
+
+        cls.booth_2 = cls.env['event.booth'].create({
+            'name': 'Test Booth 2',
+            'booth_category_id': cls.event_booth_category_1.id,
+            'event_id': cls.event_0.id,
+        })
+
+    @users('user_sales_salesman')
+    def test_event_booth_sale(self):
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.event_customer.id,
+            'pricelist_id': 1,
+        })
+
+        sale_order.write({
```

Comment:
Does it matter that you create and then write?
Could you put everything into the create?